checkpointctl.go

```diff
@@ -62,13 +73,24 @@ func setupShow() *cobra.Command {
 		false,
 		"Display mounts with full paths",
 	)
+	flags.BoolVar(
+		&showAll,
+		"all",
+		false,
+		"Display all additional information about the checkpoints",
+	)
 
-	return cmd
+	err := flags.MarkHidden("print-stats")
+	return cmd, err
```

Comment:
Rather than refactoring the function to return an error, I feel it would be better if we do a `log.Fatal(err)` instead here.

Reply:
I must say I am not the biggest friend for `Fatal()` :wink:. It doesn't go through the usual steps to exit the program. Not really a problem here, but I think returning an error is also something which makes sense. No need to change this again. Let's merge it as it is.

Reply:
O_o could you elaborate a little on the usual steps part? AFAIK, `log.Fatal` calls `os.Exit(1)` internally. Although I am not sure if it runs deferred functions before dying :thinking: